src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -285,21 +283,49 @@ class WorkspaceService(userInfo: UserInfo, dataSource: DataSource, containerDAO:
 
   def getACL(workspaceName: WorkspaceName): PerRequestMessage =
     dataSource inTransaction { txn =>
-      withWorkspaceContextAndPermissions(workspaceName, WorkspaceAccessLevel.Owner, txn) { workspaceContext =>
-        Try(gcsDAO.getACL(workspaceContext.bucketName, workspaceName)) match {
-          case Success(acl) => RequestComplete(StatusCodes.OK, acl)
-          case Failure(err) => RequestComplete(StatusCodes.Forbidden, s"Can't retrieve ACL for workspace ${workspaceName}: " + err.getMessage())
+      withWorkspaceContext(workspaceName, txn) { workspaceContext =>
+        asOwner(workspaceName) {
+          Try(gcsDAO.getACL(workspaceContext.bucketName, workspaceName)) match {
+            case Success(acl) => RequestComplete(StatusCodes.OK, acl)
+            case Failure(err) => RequestComplete(StatusCodes.Forbidden, s"Can't retrieve ACL for workspace ${workspaceName}: " + err.getMessage())
+          }
         }
       }
     }
 
   def updateACL(workspaceName: WorkspaceName, aclUpdates: Seq[WorkspaceACLUpdate]): PerRequestMessage =
     dataSource inTransaction { txn =>
-      withWorkspaceContextAndPermissions(workspaceName, WorkspaceAccessLevel.Owner, txn) { workspaceContext =>
-        val updateErrors = gcsDAO.updateACL(workspaceContext.bucketName, workspaceName, aclUpdates)
-        updateErrors.size match {
-          case 0 => RequestComplete(StatusCodes.OK)
-          case _ => RequestComplete(StatusCodes.Conflict, updateErrors)
+      withWorkspaceContext(workspaceName, txn) { workspaceContext =>
```

Comment:
Because someone just failed to combine the two on line 192 so I fixed it for them, but this one is one of the special ones that owners can do even when there's a lock.  I need a better name for asOwner that makes this clearer.  requireOwnerIgnoreLock, maybe.